src/components/Testimonial/Testimonial.jsx

```diff
@@ -0,0 +1,35 @@
+import PropTypes from 'prop-types';
+import Svg from '@/components/Svg';
+import clsx from 'clsx';
+import styles from './Testimonial.module.css';
+
+function Testimonial({ title, source, icon }) {
+  return (
+    <div className={styles.card}>
+      <div className={styles.card__inner}>
+        <div className={styles.card__info}>
+          <span className={styles.title}>{title}</span>
+          <span className={clsx('subtitle-01', styles.content)}>
```

Comment:
`subtitle-01` is an old typography. should be `subtitle-bold` and you might wanna update the font-weight in `.content` to 400.